packages/bpk-component-dialog/src/BpkDialog.scss

```diff
@@ -29,6 +29,12 @@
     }
   }
 
+  &--with-flare {
```

Comment:
The BEM naming here isn't correct — this isn't a modifier on an existing element, the flare is its own element. It should be `&__flare` (so that when used in code it would be `bpk-dialog__flare`)

Reply:
Sounds good I followed the with-icon as this is a dialog with flare as apposed to flare dialog as it's an addition to the dialog 🤷‍♀️

Reply:
That's different — `bpk-dialog--with-icon` is a modifier on the `BpkModal` component, but the icon itself is an element which is why it has `bpk-dialog__icon` as its class name.